### ISSUE TITLE
[Feature Request] Apple TV support

### ISSUE THREAD

**Issue by yannip1234** [rileytestut/AltStore#23]
Will Apple TV be supported, so Delta could be installed?

**Comment by thomas-alrek** [comment]
+1 for this. Delta with the Bluetooth controller support of iOS 13 would be awesome

**Comment by SylvainCecchetto** [comment]
Yes, AltStore support for tvOS would be fantastic! Once the store becomes available for third party app/devs it would allow apps like Kodi to be available on Apple TV easily!

**Comment by fabergoncalves** [comment]
I'll be waiting for that! It would be really great to have it on the Apple TV

**Comment by francescofact** [comment]
I also would like tvOS support!

**Comment by QuarterSwede** [comment]
I’d love to see AltStore on tvOS too. It would become the easiest way to play retro games for many.

**Comment by D-e-n-t** [comment]
AltStore shows my AppleTV in the list of install targets, but it looks like it fumbles the certificate request for tvos apps.

**Comment by bentsionmp** [comment]
Would love support for Appel TV

**Comment by jkcoxson** [comment]
Apple TV support is available for beta users, how do you install it on an Apple TV? I'm a beta user and can't figure it out.

**Comment by jkcoxson** [comment]
https://twitter.com/altstoreio/status/1339697791690563588

**Comment by adrianmihalko** [comment]
I have beta as well, but I get error:

<img width="546" alt="Screenshot 2021-03-21 at 15 10 18" src="https://user-images.githubusercontent.com/733361/111908309-0412de80-8a59-11eb-8ff4-ee339d90f357.png">

**Comment by jkcoxson** [comment]
You need to build an Apple TV specific app, you can't just side load an iOS app.

**Comment by adrianmihalko** [comment]
I get this by Install Altstore option... it is not needed on Apple TV?

**Comment by jkcoxson** [comment]
You can only side load on TV right now, there is no store yet/ever

**Comment by adrianmihalko** [comment]
Cool, thanks for the info. I'll try to install Kodi.

**Comment by sy6sy2** [comment]
Sideloading Kodi on iOS or tvOS with AltStore will only work when AltStore will be able to sign framework binaries 😕
See https://github.com/rileytestut/AltSign/pull/1

**Comment by CoderTonyChan** [comment]
Apple should build a safari app on Apple TV

**Comment by sy6sy2** [comment]
You can enable it if you jailbreak.

**Comment by alexchandel** [comment]
Does AltStore support tvOS yet?   Many sideloadable apps support tvOS now, including Provenance.  They need autoupdating.

I note that AltServer appears to find Apple TV's paired to one's developer account.  However, installing AltStore fails if the Apple TV is not paired to the computer (which is often difficult).  AltServer just needs a way to pair an Apple TV like Xcode does.
